src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -101,7 +101,6 @@ public class JwtSecurityTokenHandler : SecurityTokenHandler
         /// </summary>
         static JwtSecurityTokenHandler()
         {
-            LogHelper.LogVerbose("Assembly version info: " + LogHelper.MarkAsNonPII(typeof(JwtSecurityTokenHandler).AssemblyQualifiedName));
```

Comment:
Did we move this somewhere else? Or do we say: not needed? Is it line 438 of LogHelper?

Reply:
I think we said this wasn't needed and info is captured elsewhere now.